lib/link.js

```diff
@@ -10,7 +10,27 @@ const link = (directories, cwd = process.cwd()) => {
 
     fs.exists(dist, (exists) => {
       if (exists) { return; }
-      fs.symlinkSync(src, dist, 'dir');
+
+      fs.stat(cwdSrc, (err, stats) => {
+        if (err) {
+          if (err.code === 'ENOENT') {
+            throw new Error(`No entry: ${cwdSrc}`);
+          }
+          throw err;
+        }
+
+        const type = stats.isFile() ?
```

Comment:
Multiple ternary operators are quite hard to understand.
What do you thing about a function `getTypeByState` in outer scope that works with early returns?
``` javascript
function getTypeByStats(stats) {
  if (stats.isFile()) {
  	return 'file';
  }
  
  if (stats.isDirectory()) {
  	return 'dir';
  }

  throw new Error('Unknown entry type');
}
```